opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -706,6 +706,7 @@ namespace Opm {
                 if (has_solvent_) {
                     double& ss = priVars[Indices::solventSaturationIdx];
                     ss -= satScaleFactor * dss;
+                    ss = std::min(std::max(ss, 0.0),1.0);
```

Comment:
I guess this change is unrelated to this PR and might change the solution?

Reply:
Yes, but currently we don't test flow_ebos_solvent. So this line is not the reason for the test failure.